js/components/Request.tsx

```diff
@@ -109,7 +121,7 @@ class Request extends React.Component<IRequestProps, Partial<IRequestState>> {
         let request = null;
         if (!this.state.isCollapsed && !this.state.isReplaying && !this.state.response) {
             request = <div className="pending-request-json">
-                <pre>{JSON.stringify(this.props.request, null, 4)}</pre>
+                <pre>{JSON.stringify(this.props.request.request, null, 4)}</pre>
```

Comment:
Can we rename or separate the request prop?

Reply:
Done